examples/crm/src/sales/TransferAdminRoleModal.tsx

```diff
@@ -0,0 +1,114 @@
+import Alert from '@mui/material/Alert';
+import Autocomplete from '@mui/material/Autocomplete';
+import Button from '@mui/material/Button';
+import CircularProgress from '@mui/material/CircularProgress';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import Skeleton from '@mui/material/Skeleton';
+import { default as MuiTextField } from '@mui/material/TextField';
+import { useState } from 'react';
+import {
+    useDataProvider,
+    useGetIdentity,
+    useGetList,
+    useRefresh,
+} from 'react-admin';
+import { useNavigate } from 'react-router';
+import { USER_STORAGE_KEY } from '../authProvider';
+import { CustomDataProvider } from '../dataProvider';
+import { Sale } from '../types';
+type TransferAdminRoleModalProps = {
+    open: boolean;
+    onClose(): void;
+};
+
+export function TransferAdminRoleModal({
+    open,
+    onClose,
+}: TransferAdminRoleModalProps) {
+    const { identity } = useGetIdentity();
+    const dataProvider = useDataProvider<CustomDataProvider>();
+    const refresh = useRefresh();
```

Comment:
no, use the `refetch` from `useGetList` instead

Reply:
Since the current user is no longer an administrator once they transfered the role, I am not sure that I can only refresh the page, I need to refresh its permissions as well